src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Queries/Predicates/PredicateQuery.cs

```diff
@@ -127,15 +127,15 @@ public string GetColumnName(string propertyPath)
                     {
                         // Switch the given alias in the path with the mapped alias.
                         // aliasPart.alias -> AliasPartIndex.Alias
-                        return Dialect.QuoteForTableName($"{tableAlias}", _configuration.Schema) + "." + Dialect.QuoteForColumnName(columnName);
+                        return $"{Dialect.QuoteForAliasName(tableAlias)}.{Dialect.QuoteForColumnName(columnName)}";
                     }
                 }
                 else
                 {
                     // no property provider exists; hope sql is case-insensitive (will break postgres; property providers must be supplied for postgres)
                     // Switch the given alias in the path with the mapped alias.
                     // aliasPart.Alias -> AliasPartIndex.alias
-                    return Dialect.QuoteForTableName($"{tableAlias}", _configuration.Schema) + "." + Dialect.QuoteForColumnName(values[0]);
+                    return $"{Dialect.QuoteForAliasName(tableAlias)}.{Dialect.QuoteForColumnName(values[0])}";
```

Comment:
The main problem here is that since this PR removes the original logic for specifying the schema, 
can we be sure that the `tableAlias` were created with the schema specified?

Reply:
假如我们当前使用的租户  使用SqlServer 同时 指定了架构  `tenant1`


```SQL

-- The correct SQL would look like this
SELECT Value FROM [tenant1].[TextFieldIndex] textField where textField.[ContentType]='MyType'

-- But would the current code produce this result?
SELECT Value FROM [TextFieldIndex] textField where textField.[ContentType]='MyType'

```

Reply:
@hyzx86 Looks like he's right, my suspicion was unfounded...